tests/test_models_qcnn.py

```diff
@@ -303,3 +305,47 @@ def test_qcnn_training():
     predictions.append(1)
     labels = np.array([[1], [-1], [1]])
     test_qcnn.Accuracy(labels, predictions)
+
+
+def test_two_qubit_ansatz():
+    c = Circuit(2)
+    c.add(gates.H(0))
+    c.add(gates.RX(0, 0))
+    c.add(gates.CNOT(1, 0))
+    test_qcnn = QuantumCNN(4, 2, 2, twoqubitansatz=c)
+
+
+def test_two_qubit_ansatz_training():
```

Comment:
Same comment of before (in general, consider this comment valid for all the tests here).